src/AppInstallerCLICore/Workflows/ShowFlow.cpp

```diff
@@ -97,7 +97,7 @@ namespace AppInstaller::CLI::Workflow
             {
                 if (!agreement.Label.empty())
                 {
-                    info << Execution::ManifestInfoEmphasis << agreement.Label << " ";
+                    info << Execution::ManifestInfoEmphasis << agreement.Label << ": ";

```

Comment:
If possible, can you add the colons to source agreement labels too? I think I followed the package agreements style when doing source agreements work

Reply:
Added.